src/main/java/spaceinvaders/spaceinvaders_framework/AlienType1.java

```diff
@@ -1,5 +1,10 @@
 package spaceinvaders.spaceinvaders_framework;
```

Comment:
Why is this class/file in the 'Alien' package and in 'spaceinvaders_framework' ?